package.json

```diff
@@ -50,7 +50,7 @@
     "express": "4.15.x",
     "forever": "0.15.x",
     "govuk_frontend_toolkit": "6.0.x",
-    "govuk_template_mustache": "0.22.x",
+    "govuk_template_mustache_inheritance": "https://github.com/jonheslop/govuk_template/releases/download/v0.22.4/mustache_inheritance_govuk_template-0.22.3.tgz",
```

Comment:
to be honest I didn't fully understand this fully either. But @jonheslop seemed to have had a chat with GOV.UK folks about this and they have confirmed that this is the right way for now.

Although, I still cannot justify why its in his personal repo (just realised when you mentioned it). He could have easily done this some where in `alphagov`. I'll chase it with him when he's back